docs/docs/dev/documentation/health.md

```diff
@@ -1,13 +1,25 @@
 # Health checks
 
-Health is an extremely important part of the microservice architecture, load balancers rely on the 
-health status of the service while picking endpoint from load balancer set. Also, users want the service state to be 
-observable through the GUI or CLI. Orchestrators like Kubernetes also want to know the service status to manage 
-lifecycle of the containers. 
-
-Kuma supports several aspects of the health checking. There are two policies which allows configuring active and passive
-health checks:
-[Health Check](../../policies/health-check) and [Circuit Breaker](../../policies/circuit-breaker).
+Health is an important aspect of a microservice architecture. Load balancers use the health status
+of a service to select an endpoint. Orchestrators, such as Kubernetes, use service status
+to manage container lifecycles. Also, users want the service state to be observable through the GUI or CLI.
+
+Kuma supports several health checking methods:
+
+* [Health Check](../../policies/health-check) Policy
+
+  An **active** Kuma policy which configures a dataplane proxy to send extra traffic
+  to other dataplane proxies in order to evaluate their health.
+
+* [Circuit Breaker](../../policies/circuit-breaker) Policy
+
+  A **passive** Kuma policy which configures a dataplane proxy to monitor its existing
+  mesh traffic in order to evaluate dataplane health.
+
+* [Kubernetes](#kubernetes-probes) and [Universal](#universal-probes) Service Probes
```

Comment:
Feels like this bit of doc is mixing 2 things:

1. Showing the things users can do to manage service failures
2. Explaining how to use probes

Should this page be "dataplane failure management" (there must be a better name) and we'd move the probe description to policies?

Reply:
@lahabana Yes. I originally had a comment about this in the summary, but ended up deleting it. At the very least calling this page "Health Check" is confusing because that is the proper name of one of our policies. I agree, we should divide out the probe config, and expand on the summary.